mock-app-verifier/programs/verifier/src/processor.rs

```diff
@@ -21,16 +21,23 @@ impl Config for TransactionsConfig {
     const ID: Pubkey = pubkey!("Fg6PaFpoGXkYsidMpWTK6W2BeZ7FEfcYkg476zPFsLnS");
 }
```

Comment:
do we even need the TransactionConfig anymore?
Maybe you can eliminate it in the refactor pr.

Reply:
I am fine with that let's just define the const generics as constants in lib.rs so that it's easy to see what they are (more important for other const generics than program id).
Doesn't have to be this pr though.

Reply:
OK, it's not possible to get rid of `Config` completely, `Pubkey` can't be `const`. Let me just strip down the `Config` just to ID.